contentcuration/contentcuration/frontend/shared/views/StudioLargeLoader.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+
+  <div class="studio-large-loader">
+    <KCircularLoader :size="70" />
+    <div class="loader-text">
+      <slot ></slot>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import KCircularLoader from 'kolibri-design-system/lib/loaders/KCircularLoader.vue';
+
+  export default {
+    name: 'StudioLargeLoader',
+    components: { KCircularLoader },
+  };
+
+</script>
+
+
+<style scoped>
+
+  .studio-large-loader {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    min-height: 180px;
+  }
```

Comment:
Simplify this css class in light of https://github.com/learningequality/studio/pull/5388/files#r2388418039